### PR TITLE
impl Debug for LeakyBucket, Inner

### PR DESCRIPTION
**Pull request by kosta**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -295,6 +295,17 @@ struct Inner {
     task_tx: mpsc::Sender<Task>,
 }
 
+impl std::fmt::Debug for Inner {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "LeakyBucket {{ tokens: {}, max: {}, refill_interval: {:?}, refill_amount: {} }}",
```
Comment (udoprog) [inline src/lib.rs]: I'd recommend making use of [`debug_struct`](https://doc.rust-lang.org/std/fmt/struct.Formatter.html#method.debug_struct), it should give the same result.

Comment (kosta) [reply src/lib.rs]: TIL :)

Will fix.

**Comment by udoprog** [comment]
Thanks!

**Comment by kosta** [comment]
Thanks for merging!

Would you be so kind an push a release to crates.io? Then I can `#[derive(Debug)]` :)

**Comment by udoprog** [comment]
Published `0.7.2`. Thanks.

**Pull request merged**